src/components/component-playground.jsx

```diff
@@ -198,11 +206,17 @@ module.exports = React.createClass({
   },
 
   _renderPreview: function() {
-    return <div ref="previewContainer"
-                key="previewContainer"
-                className={this._getPreviewClasses()}>
-      {this.loadChild('preview')}
-    </div>
+    store.dispatch(changeFixture(this.state.fixtureContents.reduxStore));
+
+    return <Provider key="provider" ref="provider" store={store}>
+      {
+        () => <div ref={(ref) => this.refs.previewContainer = ref}
```

Comment:
Why using "fat-arrow"? Please use standard function() {} because there's no `es6` in this document.